cea/resources/radiation_arcgis/radiation.py

```diff
@@ -366,6 +368,10 @@ def calculate_sunrise(year_to_simulate, longitude, latitude):
     print('complete calculating sunrise')
     return sunrise
 
+def is_dst(zonename):
```

Comment:
this function is not used anywhere...

Reply:
also, it doesn't have a comment? i'm just going to assume it returns a boolean, `True`, if `zonename` is wait... what?

Reply:
done